scte35/segmentationdescriptor.go

```diff
@@ -247,7 +247,7 @@ func (d *segmentationDescriptor) CanClose(out SegmentationDescriptor) bool {
 // Equality in this sense means that two "in" events are duplicates
 // A lot of debate went in to what actually constitutes a "duplicate".  We get
 // all sorts of things from different providers/transcoders, so in the end, we
-// just settled on PTS and segmentation type
+// just settled on PTS, event id, and segmentation type
```

Comment:
Initially we were ensured that a signal could be determined unique by the TypeID and EventID.  Real life proved that to be untrue (certain networks treat EventID differently), so we moved entirely to a TypeID/PTS time existence.  That being said, these changes look safe to me and won't be tripped up by a provider duplicating EventID across multiple events.